pyproject.toml

```diff
@@ -73,7 +73,7 @@ ignore = [
     "PLR1714", # consider-using-in
     "PLR5501", # else-if-used
 ]
-fixable = ["ALL"]
+lint.fixable = ["ALL"]
 exclude = ["pynxtools/definitions/*"]
```

Comment:
I think this exclude can also be removed

Reply:
Sorry, you are right, I was too fast with the copy. Will change.

Reply:
Nothing to be sorry about. I think I actually forgot this when I created this file as this is not changed by your PR.